.github/release.yml

```diff
@@ -12,10 +12,12 @@ changelog:
     - title: Enhancements 🎉
       labels:
         - enhancement
-        - "*"
     - title: Security Hardening 🔒
       labels:
         - security-hardening
     - title: Bug Fixes 🐛
       labels:
         - bug-fix
+    - title: Other Changes
+      labels:
+        - "*"
```

Comment:
There was an incorrect rebase that included some things from main branch as new commits